docs/guides/rendering.md

```diff
@@ -29,9 +29,9 @@ function renderNode(props) {
 }
 ```
 
-> 🤖 Be sure to mix in `props.attributes` and render `props.children` in your node components! The attributes required for utilities like Slate's `findDOMNode`, and the children are the actual text content of your nodes.
+> 🤖 Be sure to mix in `props.attributes` and render `props.children` in your node components! The attributes are required for utilities like Slate's `findDOMNode`, and the children are the actual text content of your nodes.
 
-Or course, you don't have to use simple HTML elements, you can use your own custom React components too:
```

Comment:
I assume it should say "Of course", but I opted to remove it instead of fixing it, since I generally don't like phrases like "of course", "obviously" etc. in guides like this. It easily makes you feel a bit stupid as a beginner. Other than that, I think the general tone and language was absolutely on point, great work!